src/main/java/com/ithillel/service/appcontext/InjectorApplicationContext.java

```diff
@@ -0,0 +1,55 @@
+package com.ithillel.service.appcontext;
+
+import com.ithillel.service.annotation.CustomBean;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class InjectorApplicationContext implements ApplicationContext {
+
+    private static Map<String, Object> beans = new HashMap<>();
+
+    public InjectorApplicationContext() throws ClassNotFoundException {
+        List<Class<?>> classes = addClasses("src\\main\\java\\com\\ithillel");
+        for (Class<?> c : classes) {
+            if (c.isAnnotationPresent(CustomBean.class)) {
+                String name = c.getName();
+                System.out.println(name.substring(name.lastIndexOf('.') + 1));
+                beans.put(name.substring(name.lastIndexOf('.') + 1), c);
+            }
+        }
+    }
+
+    @Override
+    public Object getBean(String name) {
+        return beans.get(name);
+    }
+
+    private List<Class<?>> addClasses(String path) throws ClassNotFoundException {
+        File directory = new File(path);
+        List<Class<?>> classes = new ArrayList<>();
+        if (!directory.exists()) {
```

Comment:
if directory doesn't exist
does it make sense to move forward?
maybe you can throw exception, or just return emptylist